### PR TITLE
========= Update Release 3: Ian's Gotta Pee =========

### PR DESCRIPTION
**Pull request by SpadesNeil**
Since @Novacat went through the absolute clusterfuck of PR backlog, we can now update the server again.

But first we must review the new code.

All 400,000 lines of it.

[screams]

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Holy fuck we went from 7 pages to 3. :U

**Comment by Novacat** [comment]
Phase one is now complete. Next phase will be fixing all of the runtimes, and as many bugs as possible.

**Comment by SpadesNeil** [comment]
Capture

https://www.youtube.com/watch?v=lHKhBA3MHdk

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/datum/proc/CanProcCall(procname)
```
Comment (SpadesNeil) [inline code/datums/datumvars.dm]: Here is where I left off. Nothing funky so far that I noticed.

**Comment by Heroman3003** [comment]
https://pastebin.com/i3F9ZYet

Leaving the changelog for this update here in case anyone wonders about list of changes. I try to update as new stuff is merged, as well.

**Comment by SpadesNeil** [comment]
Github choked out and crashed when trying to review the rest of the code. Let's just hope we can trust our new dev. ¯\_(ツ)_/¯

**Pull request merged**